hack/jenkins/job-builder-image/Makefile

```diff
@@ -0,0 +1,5 @@
+TAG = 1
```

Comment:
minor nit: I think most of our tags have either followed some sort of 0.N or vN convention.